ofctrl/ofSwitch.go

```diff
@@ -165,40 +153,41 @@ func (self *OFSwitch) IsReady() bool {
 }
 
 // Handle switch connected event
-func (self *OFSwitch) switchConnected() {
-	self.changeStatus(true)
-
-	// Send new feature request
-	self.Send(openflow15.NewFeaturesRequest())
```

Comment:
Is it removing a useless request the lib sent before?

Reply:
Yes, NewFeaturesRequest is sent in Controller (ofctrl.go) when receiving "hello" from Switch.